core/src/main/scala/org/apache/spark/storage/BlockManagerMasterActor.scala

```diff
@@ -522,7 +523,9 @@ private[spark] class BlockManagerInfo(
         logInfo("Removed %s on %s on tachyon (size: %s)".format(
           blockId, blockManagerId.hostPort, Utils.bytesToString(blockStatus.tachyonSize)))
       }
+      return BlockStatus(storageLevel, 0, 0, 0)
     }
+    null
```

Comment:
actually the code should not enter into this line...unless something wrong happened in updating https://github.com/CodingCat/spark/blob/SPARK-3957/core/src/main/scala/org/apache/spark/storage/BlockManagerMasterActor.scala#L457

maybe I can change https://github.com/CodingCat/spark/blob/SPARK-3957/core/src/main/scala/org/apache/spark/storage/BlockManagerMasterActor.scala#L373 to an Option, or add a if guard in https://github.com/CodingCat/spark/blob/SPARK-3957/core/src/main/scala/org/apache/spark/storage/BlockManagerMasterActor.scala#L373